apps/heft/src/plugins/TypeScriptPlugin/Eslint.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import * as crypto from 'crypto';
+import * as TEslint from 'eslint';
```

Comment:
Can you somehow measure how much time is spent parsing TypeScript versus running rules? I believe ESLint collects metrics and exposes them  via some API. (?)

Reply:
This might help: https://github.com/eslint/eslint/blob/ec105b24f7e036ecdc4267f018529ac3765e29d5/lib/linter/timing.js#L42

Reply:
That module allows for rule duration logging. I've patched into that, but we'll need to do more research if we want to log more specific metrics.